src/runtime/tests/arc-test.ts

```diff
@@ -718,17 +727,18 @@ describe('Arc ' + storageKeyPrefix, () => {
     const storageKey = storageKeyPrefix + id.toString();
     const arc = new Arc({id, storageKey, loader, context: manifest});
 
-    const varStore = await arc.createStore(dataClass.type, undefined, 'test:0') as SingletonStorageProvider;
+    const varStore = await arc.createStore(dataClass.type, undefined, 'test:0');
     const colStore = await arc.createStore(dataClass.type.collectionOf(), undefined, 'test:1') as CollectionStorageProvider;
     const bigStore = await arc.createStore(dataClass.type.bigCollectionOf(), undefined, 'test:2') as BigCollectionStorageProvider;
+    const varHandle = await singletonHandleForTest(arc, varStore);
 
     // TODO: Reference Mode: Deal With It (TM)
     varStore.referenceMode = false;
     colStore.referenceMode = false;
 
     // Populate the stores, run the arc and get its serialization.
     // TODO: the serialization roundtrip re-generates keys using the entity ids; we should keep the actual keys
-    await getSingletonHandle(varStore).set(new dataClass({value: 'v1'}));
+    await varHandle.set(new dataClass({value: 'v1'}));
     await colStore.store({id: 'i2', rawData: {value: 'v2', size: 20}}, ['i2']);
```

Comment:
can colStore be migrated to colHandle as well?

Reply:
These didn't work for some reason. It looked complicated :-/